### PR TITLE
[W.I.P.]Spring boot program

### PR DESCRIPTION
**Pull request by dezuz**

### CODE REVIEW

File: src/main/java/com/mateacademy/controller/ApplicationController.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.servlet.ModelAndView;
+
+@RestController
+public class ApplicationController {
+
+    @Autowired
```
Comment (Taniann) [inline src/main/java/com/mateacademy/controller/ApplicationController.java:15]: better use constructor di

File: src/main/java/com/mateacademy/controller/UserController.java [hunk 2]
```diff
@@ -0,0 +1,72 @@
+package com.mateacademy.controller;
+
+import com.mateacademy.model.User;
+import com.mateacademy.service.UserServiceImpl;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
```
Comment (Taniann) [inline src/main/java/com/mateacademy/controller/UserController.java]: dont use wild card imports

File: src/main/java/com/mateacademy/model/BaseModel.java [hunk 3]
```diff
@@ -0,0 +1,22 @@
+package com.mateacademy.model;
+
+import lombok.Getter;
+import lombok.Setter;
+import lombok.experimental.Accessors;
+
+import javax.persistence.*;
```
Comment (Taniann) [inline src/main/java/com/mateacademy/model/BaseModel.java]: the same about wild card import

File: src/main/java/com/mateacademy/service/ProducerServiceImpl.java [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+import com.mateacademy.repository.ProducerRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class ProducerServiceImpl implements ProducerService{
+
+    @Autowired
```
Comment (Taniann) [inline src/main/java/com/mateacademy/service/ProducerServiceImpl.java:13]: better use constructor di

File: src/main/java/com/mateacademy/service/UserServiceImpl.java [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+import java.util.HashSet;
+import java.util.List;
+
+@Service
+public class UserServiceImpl implements UserService {
+
+    private UserRepository userRepository;
+    private RoleRepository roleRepository;
+
+    @Autowired
```
Comment (Taniann) [inline src/main/java/com/mateacademy/service/UserServiceImpl.java:20]: you can use lombok annotations for these purpose

**Review by Taniann — CHANGES_REQUESTED** [review]
please resolve comments

File: src/main/java/com/mateacademy/controller/ApplicationController.java (hunk 1 above)
Comment (dezuz) [reply src/main/java/com/mateacademy/controller/ApplicationController.java:15]: For me, it's better to use @Autowired

File: src/main/java/com/mateacademy/configuration/SecurityConfig.java [hunk 6]
```diff
@@ -0,0 +1,66 @@
+//package com.mateacademy.configuration;
```
Comment (Taniann) [inline src/main/java/com/mateacademy/configuration/SecurityConfig.java]: remove it from your PR

**Pull request merged**